src/WebJobs.Script/Description/DotNet/DynamicFunctionAssemblyLoadContext.cs

```diff
@@ -6,6 +6,7 @@
 using System.Reflection;
 using System.Runtime.Loader;
 using System.Text;
+using Microsoft.Azure.WebJobs.Script.Abstractions.Description;
```

Comment:
I know @mathewc was in favor the namespace renames, but I'm wondering if that's something we should really do. The impact was quite significant and many of these types were public.

Reply:
@fabiocav, other than core-tools do you think anyone else depends on it? Though, I realize this is public and we can't control or know of who uses it.

Reply:
Well, even moving the Types to a different assembly is a breaking change, so we've already decided on that right? I'm not aware of anyone consuming these types but us, but I see your point.

Reply:
Missed updating this, but adding `Abstractions` to the namespace here doesn't make a lot of sense and doesn't really follow the convention set in other projects. These are just grouped in the same package/assembly, but should follow the appropriate namespace for the area.